### ISSUE TITLE
Add Wright's taxonomy to the main Syriaca taxonomy and/or provide crosswalks

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary-data#64]
We would want to include Wright's genre terms taxonomy in the broader Syriaca taxonomy. Since the BL catalogue is going to be a separate publication, we can keep using Wright's for the BL data (and maybe define it somewhere more formally in our documentation pages, which probably needs a separate issue). This taxonomy should be crosswalked to the Syriaca taxonomy (e.g., Wright''s term for biblical texts should be mapped as a skos:closeMatch to a Syriaca one -- and new terms minted there if needed).

Declaring Wright's taxonomy may mean that we need to update the references in the TEI data, but that will be a separate issue pending our decisions here.

**Comment by davidamichelson** [comment]
Yes, in theory :)

**Comment by wlpotter** [comment]
Priority: create the documentation page for the Wright taxonomy and updating references in data.

Wait on the crosswalk

To do here: split into two issues, one for documenting the taxonomy from Wright; another (for later) to crosswalk with Syriaca taxonomy.
